src/inkbird_ble/parser.py

```diff
@@ -16,6 +16,8 @@
 from home_assistant_bluetooth import BluetoothServiceInfo
 from sensor_state_data import SensorLibrary
 
+IAMT1_SERVICE_UUID = "0000ffe4-0000-1000-8000-00805f9b34fb"
```

Comment:
`0000ffe4` will match other devices as well as `0000-1000-8000-00805f9b34fb` is the base part of the base UUID https://github.com/Bluetooth-Devices/bluetooth-data-tools/blob/8fdc7cade9c3b87dfdd77cb741d1fd83fbc154ea/src/bluetooth_data_tools/gap.py#L8